src/credentials.ts

```diff
@@ -309,27 +310,61 @@ export class CredentialIssuer extends CredentialKeeper {
   }
 
   /**
-   * Creates and signs a Credential, and commits it to storage.
+   * Creates and signs a Credential.
    *
    * @param credParams - credential attributes
    * @returns SignedCredential instance
+   * @throws Error on credential claim with 'null' or 'undefined' value
    * @category Credential Management
    */
   async create<T extends BaseMetadata>(credParams: ISignedCredCreationArgs<T>) {
```

Comment:
In context of the `ObjectKeeper`-based classes, the `create` method is part of the CRUD interface and is expected to be making use of the storage.

So `this.persist` would  be used here rather than inside `issue` (which is not part of the `ObjectKeeper` interface at all)